src/Card/Card.js

```diff
@@ -18,8 +19,9 @@ const Card = ({ title, snippet }) => {
     <div className="card">
         {/* <img src={star} alt="Star Icon" className="star-icon"/> */}
         <div className="card-content">
-      <h2 onClick={handleTitleClick}> Article Title: {title}</h2>
-      <p>Snippet: {snippet}</p>
+        <p
+          dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(snippet) }}
```

Comment:
I would love nicks input on dangerouslysetinnerhtml as well dow nthe road because its a fascinating thing that I only came across earlier in my work for this project. I would love if this was the end all solution because it seems like a snappy fix